Source/VolunteerReporting/Read/CaseReportsForListing/CaseReportForListing.cs

```diff
@@ -0,0 +1,29 @@
+using Concepts;
+using System;
+
+namespace Read.CaseReportsForListing
+{
+    public class CaseReportForListing
+    {
+        public Guid Id { get; private set; }
+        public CaseReportStatus Status { get; internal set; }
+        public DataCollectorId DataCollectorId { get; internal set; }
+        public string DataCollectorDisplayName { get; internal set; } = "Unknown"; //QUESTION: Should this be a concept with default value if unknown?
```

Comment:
How will this work with localization in the future? We can either localize the default value in the read model or should frontend be in charge of retrieving the correct value based on the users language?

Reply:
Good point. What you said 😊 I have no solution atm